### PR TITLE
KIALI-502 make sure summary panelresets when graph is refreshed

### PR DESCRIPTION
**Pull request by jshaughn**
note - also includes a fix for %error calc in summary panel rps chart

### CODE REVIEW

File: src/components/SummaryPanel/RpsChart.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export class RpsChart extends React.Component<RpsChartTypeProp, {}> {
       maxRps = sample > maxRps ? sample : maxRps;
       sumRps += +sample;
     }
-    const avgRps = len === 0 ? 0 : sumRps / (len - 1);
+    const avgRps = len < 2 ? 0 : sumRps / (len - 1);
```
Comment (cfcosta) [inline src/components/SummaryPanel/RpsChart.tsx]: Minor, but I think this fix should be on a different commit so it can be referenced/queried if necessary.

Comment (jshaughn) [reply src/components/SummaryPanel/RpsChart.tsx]: Guilty as charged, I have a [bad] habit of including tweaks.  I will try to be better.
